hedera-node/hedera-schedule-service-impl/src/main/java/module-info.java

```diff
@@ -1,25 +1,24 @@
-import com.hedera.node.app.service.schedule.impl.ScheduleServiceImpl;
-
 module com.hedera.node.app.service.schedule.impl {
-    requires transitive com.hedera.node.app.service.mono;
     requires transitive com.hedera.node.app.service.schedule;
     requires transitive com.hedera.node.app.spi;
     requires transitive com.hedera.node.hapi;
     requires transitive com.hedera.pbj.runtime;
     requires transitive dagger;
     requires transitive javax.inject;
-    requires com.swirlds.jasperdb;
+    requires com.hedera.node.app.service.mono;
+    requires com.hedera.node.app.service.token;
+    requires com.hedera.node.config;
+    requires com.swirlds.common;
+    requires com.swirlds.config;
+    requires com.swirlds.virtualmap;
+    requires org.apache.commons.lang3;
```

Comment:
@rbair23 @OlegMazurov @jasperpotts I thought we wanted to eliminate the use of the Apache Lang3 library across the board. Should we be really adding yet another usage of it to production code?

Reply:
Not sure what brought that in, will try removing it.

Reply:
Removed.